stock_picking_show_return/README.rst

```diff
@@ -0,0 +1,51 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=================================
+Display returns on stock pickings
+=================================
+
+This module adds one tab in the pickings form view to display the returns
+pickings related to the current one.
+
+Usage
+=====
+
+#. Go to any picking that has a return
+#. See the new tab "Returns" appears.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/154/11.0
```

Comment:
Should be `12.0`

Reply:
You can use README by fragments and generate this automatically: https://github.com/OCA/maintainer-tools/#readme-generator

Reply:
Done